README.md

```diff
@@ -542,3 +548,60 @@ And run the tests from the root of the source code:
 ```text
 bundle exec rake parallel_spec
 ```
+
+See also `.travis.yml` for information on running the acceptance and other tests.
+
+### Migration path to v7.0.0
+
+As of `v7.0.0` of this module a major rework has been done to adopt the [puppet-resource_api](https://github.com/puppetlabs/puppet-resource_api) into the module and use it style of code in place of the original form of Puppet Type and Providers. As part of this several breaking changes where made to the code that will need to be accounted for whenever you update to this new version of the module.
```

Comment:
looks god to me. Maybe one sentence why this change is made?

Reply:
Updated to include the primary reasoning